modules/apps/site/site-cms-site-initializer/src/main/java/com/liferay/site/cms/site/initializer/internal/display/context/BaseSectionDisplayContext.java

```diff
@@ -180,10 +180,12 @@ public List<FDSActionDropdownItem> getFDSActionDropdownItems() {
 			new FDSActionDropdownItem(
 				StringBundler.concat(
 					themeDisplay.getPathFriendlyURLPublic(),
-					GroupConstants.CMS_FRIENDLY_URL, "/f/{embedded.id}"),
-				"pencil", "view",
+					GroupConstants.CMS_FRIENDLY_URL, "/e/view-folder/",
```

Comment:
why use the /e/view-folder/ ? /f/ is the friendlyURL configured for folders, but you need to have the display page market as default.

Reply:
@marco-leo We have more than one display page for folders (one for navigation and one for edition), so we address them explicitly.